xbmc/platform/win10/filesystem/WinLibraryDirectory.cpp

```diff
@@ -183,21 +179,21 @@ bool CWinLibraryDirectory::Remove(const CURL& url)
       return false;
   try
   {
-    Wait(folder->DeleteAsync(StorageDeleteOption::PermanentDelete));
+    Wait(folder.DeleteAsync(StorageDeleteOption::PermanentDelete));
     exists = true;
   }
-  catch(Platform::Exception^ ex)
+  catch(winrt::hresult_error const& ex)
   {
-    std::string error = FromW(std::wstring(ex->Message->Data()));
+    std::string error = FromW(std::wstring(ex.message().c_str()));
```

Comment:
the extra `std::wstring` shouldn't be necessary here and in other similar places.

Reply:
thanks